cmd/fleetctl/get.go

```diff
@@ -184,6 +187,15 @@ func (eacp enrichedAppConfigPresenter) MarshalJSON() ([]byte, error) {
 			eacp.Vulnerabilities,
 		},
 	})
+	if err != nil {
+		return nil, err
+	}
+
+	// we need to marshal and combine both groups separately because
+	// enrichedAppConfig has a custom marshaler.
```

Comment:
Looks like we have something like this in 3 places, might be worth extracting a helper func, especially if it adds the checks I mention in another comment.